vyxal/elements.py

```diff
@@ -4600,6 +4601,7 @@ def zfiller(lhs, rhs, ctx):
     "k⟇": process_element("codepage", 0),
     "k½": process_element("LazyList([1,2])", 0),
     "kḭ": process_element("2 ** 32", 0),
+    "k₁": process_element("LazyList([1, 1])", 0),
```

Comment:
Should this element be included in this PR?

Reply:
Yes because I need it for an answer lol